docs/overview/architecture.md

```diff
@@ -1,44 +1,27 @@
 ---
 title: Architecture
-sidebar_position: 1
+sidebar_position: 2
 ---
 
-Quickwit has been designed from the ground up for cost-efficiency and scalability. 
-
-Quickwit core relies on two processes:
+Quickwit distributed search engine relies on 4 major services and one maintenance service:
 
 - The Searchers for executing search queries from the REST API.
 - The Indexers that index data from data sources.
+- The Metastore that stores the index metadata in a PostgreSQL-like database or in a cloud storage file.
+- The Control plane that schedules indexing tasks to the indexers.
+- The Janitor that executes periodic maintenance tasks.
 
 Moreover, Quickwit leverages existing infrastructure by relying on battled-tested technologies for index storage, metadata storage, and ingestion:
 
-### Index storage
-
-Quickwit stores its indexes's file on Amazon S3. In a single-server deployment, this can also be your local disk. We plan to support other cloud storage, as well as HDFS.
-
-### Metastore
-
-Quickwit gathers index metadata into a metastore to make them available across the cluster. Indexers push index data on the index storage and publish metadata to the metastore.
-
-In a clustered deployment, the metastore is typically a traditional RDBMS like PostgreSQL which we only support today. In a single-server deployment, it’s also possible to rely on a local file or on Amazon S3.
-
-### Source
-
-Quickwit supports multiple sources to ingest data from.
-A source can be a simple file or a complex system like a message queue.
-Additionally the indexing service will provide a zero-configuration ingest API.
-
-A file is ideal for a one-time ingestion like an initial load, a message queue is ideal to continuously feed data into the system. 
-
-Quickwit indexers connect directly to external message queues like Kafka and guarantee the exactly-once semantics. If you need support for other distributed queues, please vote for yours [here](https://github.com/quickwit-oss/quickwit/issues/1000).
+- Cloud storage like AWS S3, GCS, ABS, etc. for index storage.
+- Postgresql-like for metadata storage.
```

Comment:
We should probably just say `Postgresql` here, since `like` suggests that you can use more than just Postgres, which *technically* you can, but I'm not sure if SQLx's issues with compatible DBs like Yugabyte, etc... are fixed.